### DISCUSSION TITLE
How do I style a Modal

### DISCUSSION THREAD

**Discussion by dhope1106** [chakra-ui/chakra-ui#4364] [Q&A]
I want to create a Modal variant with and set the default width and background color (among other things). I can't find documentation that says exactly how to do it, but I figured using `variants` would be the way to go.

I've put my best attempt on Code Sandbox here: https://codesandbox.io/s/vigilant-germain-u3mkx?

Any suggestions would be welcome.

Thanks,
David

**Comment by aaronkik** [comment]
Hi David,

If you go to the component you want to customise on https://chakra-ui.com/docs/overlay/modal and click on "View theme source" you should be able to get an idea of what you need to do.

Since the modal component contains multiple "parts/components" this needs to be defined in the object as well as shown [here](https://github.com/chakra-ui/chakra-ui/blob/3205ad6d55f935f37db8f61d2dde7eaa2481578c/packages/theme/src/components/modal.ts#L3).

I've created a sandbox of what you were trying to acheive. Hope it helps :) 

https://codesandbox.io/s/wispy-snow-vuet5?file=/modal.js

**Reply by aaronkik** [reply]
Oh and you mentioned not be able to find the docs so here it is: https://chakra-ui.com/docs/theming/customize-theme#customizing-component-styles
